sdk/python/feast/data_source.py

```diff
@@ -519,18 +519,27 @@ class PushSource(DataSource):
 
     name: str
     schema: Dict[str, ValueType]
-    batch_source: Optional[DataSource]
+    batch_source: DataSource
+    event_timestamp_column: str
 
     def __init__(
         self,
         name: str,
         schema: Dict[str, ValueType],
-        batch_source: Optional[DataSource] = None,
+        batch_source: DataSource,
+        event_timestamp_column="timestamp",
```

Comment:
nit: surprised lint didn't catch this, but I feel like it should be `event_timestamp_column = "timestamp"`?

also, can you add docstring for `__init__` here?